demo/src/app/components/timepicker/demos/validation/timepicker-validation.ts

```diff
@@ -1,11 +1,9 @@
 import {Component} from '@angular/core';
-import {NGB_TIMEPICKER_DIRECTIVES} from '@ng-bootstrap/ng-bootstrap';
-import {FormControl, REACTIVE_FORM_DIRECTIVES} from '@angular/forms';
+import {FormControl} from '@angular/forms';
 
 @Component({
   selector: 'ngbd-timepicker-validation',
-  template: require('./timepicker-validation.html'),
-  directives: [NGB_TIMEPICKER_DIRECTIVES, REACTIVE_FORM_DIRECTIVES]
+  template: require('./timepicker-validation.html')
```

Comment:
All the other ones use templateUrl, but this one still uses template with require().